samples/grids/data-grid/column-types/src/DataGridColumnTypes.tsx

```diff
@@ -108,22 +108,22 @@ export default class DataGridColumnTypes extends React.Component<any, any> {
         this.grid = grid;
     }
 
-    public getProductivityChart(props: IIgrCellTemplateProps) {
-        const info = props.dataContext as IgrTemplateCellInfo;
-        return (
-            <div className="gridSparklineContainer">
-               <IgrSparkline
-                   height="30px" width="100%"
-                   displayType="Column"
-                   dataSource={info.rowItem.Productivity}
-                   valueMemberPath="Value"
-                   labelMemberPath="Week"
-                   lineThickness={2}
-                   brush="rgb(21, 190, 6)"
-                   negativeBrush="rgb(211, 17, 3)" />
-            </div>
-        );
-    }
+    // public getProductivityChart(props: IIgrCellTemplateProps) {
```

Comment:
please add comment: "TODO un-comment this code when editing templated columns is corrected"